### PR TITLE
[DOC-9058] Add --vanity-name flag to ccloud docs

### PR DESCRIPTION
**Pull request by gemma-shay**
[DOC-9058]

Ref [Slack thread](https://cockroachlabs.slack.com/archives/C01J7MGJ3S7/p1680727768830509)

Preview: [src/current/cockroachcloud/ccloud-get-started.md](https://deploy-preview-17990--cockroachdb-docs.netlify.app/docs/cockroachcloud/ccloud-get-started.html)

### CODE REVIEW

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 1]
```diff
@@ -45,11 +45,18 @@ If you are a member of more than one [CockroachDB Cloud organization](https://ww
...
+ccloud auth login --org {organization label}
 ~~~
 
 The organization label is found on the **Settings** page of the CockroachDB Cloud Console.
 
+If your organization has a custom URL, use the `--vanity-name` flag to log in:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud auth login --vanity-name {custom organization URL}
```
Comment (sjbarag) [inline src/current/cockroachcloud/ccloud-get-started.md:57]: From what I can tell, the `--vanity-name` flag doesn't support the full organization URL (`https://cockroachlabs.cloud/login/acme`), but expects just the last part after `/login/` (`acme` in this example):

https://github.com/cockroachlabs/ccloud-private/blob/49beebd962ffec81e5ccac0cf0bbb5b2d0749f90/auth/auth_login.go#L81

`ccloud auth login --help` currently provides:

```
Flags:
  -h, --help                 help for login
      --no-redirect          disable redirection to a browser for login
      --org string           organization label
      --vanity-name string   vanity name, the last part of the custom URL for an org-specific login page
```

but I'm not sure what a clean way to say "the last part of the custom URL for an org-specific login page" would be 😅

**Review by sjbarag — CHANGES_REQUESTED** [review]
Thanks @mdlinville! Generally looks good, but "URL" is a bit misleading in the placeholder since we don't support full vanity URLs at the moment. I wonder if there's a better word we can use there 🤔

File: src/current/cockroachcloud/ccloud-get-started.md (hunk 1 above)
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-get-started.md:57]:
```suggestion
ccloud auth login --vanity-name {custom-organization-name}
```

Replace `{custom-organization-name}` with the final portion of the custom sign-in URL, after `/login/`. Do not pass the full custom sign-in URL.

Comment (mdlinville) [reply src/current/cockroachcloud/ccloud-get-started.md:57]: Does my suggested edit resolve this?

Comment (sjbarag) [reply src/current/cockroachcloud/ccloud-get-started.md:57]: Yep, that works for me.

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 2]
```diff
@@ -45,11 +45,18 @@ If you are a member of more than one [CockroachDB Cloud organization](https://ww
...
+~~~
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-get-started.md:58]:
```suggestion
```

**Review by sjbarag — APPROVED** [review]
Thanks for the quick fix!

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 3]
```diff
@@ -45,11 +45,20 @@ If you are a member of more than one [CockroachDB Cloud organization](https://ww
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-ccloud auth login --org <organization label>
+ccloud auth login --org {organization label}
```
Comment (taroface) [inline src/current/cockroachcloud/ccloud-get-started.md:48]: Just for consistency with how `custom-organization-name` is hyphenated.

```suggestion
ccloud auth login --org {organization-label}
```

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 4]
```diff
@@ -45,11 +45,20 @@ If you are a member of more than one [CockroachDB Cloud organization](https://ww
...
 The organization label is found on the **Settings** page of the CockroachDB Cloud Console.
 
+If your organization has a custom URL, use the `--vanity-name` flag to log in:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud auth login --vanity-name {custom-organization-name}
+~~~
+
+Replace `{custom-organization-name}` with the final portion of the custom sign-in URL, after `/login/`. Do not pass the full custom sign-in URL.
```
Comment (taroface) [inline src/current/cockroachcloud/ccloud-get-started.md:60]:
```suggestion
Replace `{custom-organization-name}` with the portion of the custom sign-in URL that follows `/login/`. Do not pass the full custom sign-in URL.
```

**Review by taroface — APPROVED** [review]
LGTM with two small suggestions.

**Pull request merged**